examples/example_write_individual_address.py

```diff
@@ -0,0 +1,24 @@
+"""Example for writing an individual address to a KNX device."""
+import asyncio
+
+from xknx import XKNX
+from xknx.management import procedures
+from xknx.telegram import IndividualAddress
+
+
+async def main():
+    """
+    Write the individual address 1.1.3 to a device in programming mode.
+
+    This fails if multiple devices are in programming mode and/or when there is no device found in programming mode.
+    """
+    async with XKNX() as xknx:
+        individual_address = IndividualAddress("1.1.3")
+        task = asyncio.create_task(
+            procedures.nm_invididual_address_write(xknx, individual_address)
+        )
+        await task
```

Comment:
```suggestion
        await procedures.nm_invididual_address_write(xknx, individual_address)
```

Reply:
How can I inject a telegram midway during the communication without a task? Currently, the test still fails in that I cannot seem to capture the `IndividualAddressResponse` telegram.

Reply:
In a test you can use a task to inject a telegram while it is running. There are some examples in the codebase.
For an example it shouldn't be needed to do that.

Reply:
Here is an example <https://github.com/XKNX/xknx/blob/88fe96ba54f5d9da3fb1dab134d74b0fb0538dff/test/management_tests/procedures_test.py#L59>
If you need time to pass in a test you can use our time_travel helper 🚀 https://github.com/XKNX/xknx/blob/88fe96ba54f5d9da3fb1dab134d74b0fb0538dff/test/conftest.py#L48

Reply:
I added a few tests for the `nm_invididual_address_write` procedure.